CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 23.09+ (???)
 ------------------------------------------------------------------------
+- Fix: [#2152] Crash when too many messages to be shown.
```

Comment:
Could you make this follow the numerical order please?

Reply:
ah yep forgot it was descending. i'll have to do the same with my other PR